backend/src/controllers/preferences/wishlist.controller.ts

```diff
@@ -33,21 +33,16 @@ const setWishlist = async (
 };
 
 const deleteWishlist = async (
-  req: TypedRequestQuery<WishlistInput>,
```

Comment:
Why did you remove the `WishlistInput` type? :)
 
I would expect it to be declared as the `type WishlistInput = { wishlistId: string; }`